force-app/main/default/chat/lwc/messageItem/messageItem.js

```diff
@@ -0,0 +1,24 @@
+//todo:: const labels in html
+import {api, LightningElement} from 'lwc';
+
+import {MessageWrapper, MessageWrapperBuilder} from "./messageWrapper";
+export {MessageWrapper, MessageWrapperBuilder};
+
+export default class MessageItem extends LightningElement {
+    /**
+     * @type {MessageWrapper}
+     */
+    _message;
+
+    @api
+    set message(message) {
```

Comment:
I think it will work even without the get-set. Why did you add them here?

Reply:
I added getter to handle null value